src/components/book-form/book-form.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addBook } from '../../redux/actions/library';
+import './book-form.styles.scss';
+import { CATEGORIES } from '../../constants/constants';
+
+class BookForm extends React.Component {
+	constructor() {
```

Comment:
Further to my comment previously. Have you researched on the suggestion I made? About not having to use constructor here.